### PR TITLE
split single config file -> client config && server config, make examples easy to understand

### PR DESCRIPTION
**Pull request by cch123**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

**Comment by cch123** [comment]
https://github.com/mosn/mosn/issues/1158

**Comment by taoyuanyuan** [comment]
modify readme

**Comment by cch123** [comment]
> modify readme

fixed

File: examples/cn_readme/sofarpc-with-xprotocol-sample/README.md [hunk 1]
```diff
@@ -48,18 +50,29 @@ go run server.go
 
 ### 启动MOSN
 
-+ 使用config.json 运行非TLS加密的MOSN
++ 使用普通配置运行非TLS加密的MOSN
 
+启动 client 端:
 ```
-./main start -c config.json
+./main start -c server_config.json
```
Comment (taoyuanyuan) [inline examples/cn_readme/sofarpc-with-xprotocol-sample/README.md:57]: server -> client

**Review by taoyuanyuan — APPROVED** [review]
LGTM

**Pull request merged**